taskcluster/ci/config.yml

```diff
@@ -81,6 +81,12 @@ taskgraph:
             default-repository: git@github.com:mozilla-extensions/login-study
             default-ref: master
             type: git
+        webcompat:
+            name: "Web Compat"
+            project-regex: webcompat-addon$
+            default-repository: git@github.com:mozilla-extensions/webcompat-addon
```

Comment:
This should be `https://github.com/mozilla-extensions/webcompat-addon`. (The `git@` form is for private repos, and requires the private config settings.) This is why the webcompat build in the PR is hanging -- it's trying to clone, but doesn't have an ssh key to clone with.

Reply:
Ah, yeah. Fixed. :)